inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongClusterController.java

```diff
@@ -297,6 +297,13 @@ public Response<Boolean> unloadNode(@PathVariable Integer id) {
         return Response.success(clusterService.unloadNode(id, LoginUserUtils.getLoginUser().getName()));
     }
 
+    @RequestMapping(value = "/cluster/node/getManagerSSHPublicKey", method = RequestMethod.GET)
+    @ApiOperation(value = "Obtain the SSH public key from the manager to install the agent.")
+    @OperationLog(operation = OperationType.GET, operationTarget = OperationTarget.CLUSTER)
```

Comment:
Remove it.`@OperationLog` is used to save operation logs, and this interface does not require this record.